src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let sum = days * 40;
```

Comment:
Prefer using `const` instead of `let` for `sum` since it's not being reassigned in a broader scope. If you need to reassign it within the conditionals, consider using a different variable name to represent the discounted sum.